handler.go

```
@@ -204,7 +204,46 @@ func messageCreateHandler(b *DiscordBot, cid string, oai *OpenAiService) func(s
 
 func forgetCommandHandler(b *DiscordBot) func(s *discordgo.Session, i *discordgo.InteractionCreate) {
 	return func(s *discordgo.Session, i *discordgo.InteractionCreate) {
+		if i.Type != discordgo.InteractionApplicationCommand {
+			return
+		}
 		if i.ApplicationCommandData().Name == "forget" {
+			// 確認ボタンを表示
+			s.InteractionRespond(i.Interaction, &discordgo.InteractionResponse{
```

golangci-lint (2.12.2)

[error] Error return value of `s.InteractionRespond` is not checked

(errcheck)